### DISCUSSION TITLE
multiple parts in controller stl

### DISCUSSION THREAD

**Discussion by maxyurk** [juhakaup/keyboards#34] [General]
I tried to order a 3d print for jlc and got the following message for the controller parts:

> "Multiple parts are detected in your file, sorry to tell you that each file could only support 1 part for printing, please kindly split the multiple parts into single files to upload separately. Thanks !"

I don't have much experience with 3d modeling.
I'm trying to print the rp2040 4x6 variant.
I installed fusion360 and opened the `cygnus_3x6.f3z` file, tried to figure out which part I need to mark visible and which not and got lost.

Could you please point me to the items I need checked for it?
Or upload the STLs splitted if that's not too much trouble.

TIA

**Comment by juhakaup** [comment]
It seems that I haven't uploaded the fusion file for the controller holders.
I can upload the separate parts later tonight.

**Comment by juhakaup** [comment]
I've added the separate files to https://github.com/juhakaup/keyboards/tree/main/Cygnus%20v1.0/requests%20and%20other%20experiments/controller_holder_rp2040

They should actually be included in the fusion file. You just need to open the controller holder component, that should show up as a separate file. But yeah, it's not something you would just figure out by poking around.

**Reply by maxyurk** [reply]
thanks!
